### PR TITLE
feat: Add admin resume process endpoint

### LINKED ISSUE #2151: Add admin resume process endpoint
Make resume process endpoint available for admin API

`POST /admin/v1/process-instances/{processInstanceId}/resume`

### PR DESCRIPTION
**Pull request by almerico**
fixes Activiti/Activiti#2151

### CODE REVIEW

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceAdminControllerImplIT.java [hunk 1]
```diff
@@ -68,10 +73,10 @@
         ServicesRestAutoConfiguration.class,
         ServicesCoreAutoConfiguration.class})
 @ComponentScan(basePackages = {"org.activiti.cloud.services.rest.assemblers", "org.activiti.cloud.alfresco"})
-@EnableAutoConfiguration(exclude = { SecurityAutoConfiguration.class, SecurityFilterAutoConfiguration.class})
+@EnableAutoConfiguration(exclude = {SecurityAutoConfiguration.class, SecurityFilterAutoConfiguration.class})
 public class ProcessInstanceAdminControllerImplIT {
 
-    private static final String DOCUMENTATION_IDENTIFIER = "process-instance";
+    private static final String DOCUMENTATION_IDENTIFIER = "process-admin";
```
Comment (salaboy) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceAdminControllerImplIT.java]: @almerico can we call this process-instance-admin?

File: activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java [hunk 2]
```diff
@@ -341,7 +341,7 @@ public void suspendShouldPutProcessInstanceInSuspendedState() {
         assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
         return responseEntity;
     }
-
+//almer
```
Comment (salaboy) [inline activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java]: @almerico can we remove this one?

File: activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java [hunk 3]
```diff
@@ -388,6 +388,45 @@ public void resumeShouldPutASuspendedProcessInstanceBackToActiveState() {
...
+    @Test
+    public void adminResumeShouldPutASuspendedProcessInstanceBackToActiveState() {
+        //given
+        ResponseEntity<CloudProcessInstance> startProcessEntity = processInstanceRestTemplate.startProcess(processDefinitionIds.get(SIMPLE_PROCESS));
+        
+        //First suspend process
+        //testadmin should see process instances at admin endpoint
+        keycloakSecurityContextClientRequestInterceptor.setKeycloakTestUser("testadmin");
+        ResponseEntity<Void> responseEntity = adminExecuteRequestSuspendProcess(startProcessEntity);
+
```
Comment (salaboy) [inline activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java:405]: @CTI777 it wouldn't hurt to check that the process instance is SUSPENDED at this point.. just to make sure that the next check after resume (RUNNING) make sense

Comment (CTI777) [reply activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java:405]: Should be implemented

**Pull request merged**